qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -289,3 +289,120 @@ def default_values(self) -> List[DefaultCalValue]:
                     defaults.append(DefaultCalValue(value, param.name, tuple(), name))
 
         return defaults
+
```

Comment:
Let's have the policy that an init arg should not change the shape of the schedule.